LSP-Grammarly.sublime-commands

```diff
@@ -0,0 +1,13 @@
+[
+    { "caption": "LSP-Grammarly: Login to Grammarly", "command": "lsp_grammarly_execute_login"},
+    { "caption": "LSP-Grammarly: Logout from Grammarly", "command": "lsp_grammarly_execute_logout"},
+    { "caption": "LSP-Grammarly: Check if connected to Grammarly", "command": "lsp_grammarly_execute_is_connected"},
+    {
+        "caption": "LSP-Grammarly: Settings",
```

Comment:
All other LSP packages (and many non-LSP too) use a `Preferences: <package_name> Settings` convention here. That helps with user fixing this command regardless if searching for "Preferences" or "Settings".